pkgs/top-level/all-packages.nix

```diff
@@ -17793,7 +17793,9 @@ with pkgs;
   coq_8_4 = coqPackages_8_4.coq;
   coq_8_5 = coqPackages_8_5.coq;
   coq_8_6 = coqPackages_8_6.coq;
-  coq_HEAD = callPackage ../applications/science/logic/coq/HEAD.nix {};
+  coq_HEAD = callPackage ../applications/science/logic/coq {
+    version = "8.7pre-2017-05-28";
```

Comment:
Sorry, I meant `coq-unstable-2017-05-28` (version is in the date).

I see[1], otherwise it'd be hard to tell if the first rule of Coq-Club were to not talk about pre-release `coq-version`. I'd say it's still better to wait until the coqdev team have put the 8.7pre tag after (not during) the feature freeze on June 1st.
It is the way a vcs is defined that a repo state can only refer to the previous most-recent repo state, or a checkpoint, rather than a future repo state many commits away from the present.

[1] e.g. https://sympa.inria.fr/sympa/arc/coqdev/2017-05/msg00021.html